connector/src/main/java/org/geysermc/connector/network/translators/bedrock/entity/player/BedrockMovePlayerTranslator.java

```diff
@@ -155,6 +152,53 @@ public boolean isValidMove(GeyserSession session, MovePlayerPacket.Mode mode, Ve
         return true;
     }
 
+    /**
+     * Adjust the Bedrock position before sending to the Java server to account for inaccuracies in movement between
+     * the two versions.
```

Comment:
Should state somewhere in this doc that returning `null` means the movement packet will be cancelled (as documented in the `translate` method of this class.

Reply:
This is mentioned in the return value.